core/tasks.py

```diff
@@ -28,7 +28,7 @@ def _field_file_to_local_path(field_file: FieldFile) -> Generator[Path, None, No
     with field_file.open('rb'):
         file_obj: File = field_file.file
 
-        if isinstance(file_obj, S3Boto3StorageFile):
+        if not Path(file_obj.name).exists() or isinstance(file_obj, S3Boto3StorageFile):
```

Comment:
When does this case occur? When is the file not on disk but also not a `S3Boto3StorageFile`?

Reply:
This happens in the development setup.  The file is not an `S3Boto3StorageFile`, and the path is relative to some root but not the current cwd, so the path appears to not exist.

You can see this by uploading an algorithm to the workflow; when it tries to validate the docker, the file path is not complete enough to find the file.

Reply:
Yes, good point. I realize since we're using `MinioMediaStorage` in development, the file does not exist, but also is not a `S3Boto3StorageFile`.

This is a good general fix for right now, thanks.